src/Kafka/KafkaInstances/components/InstancesTable.tsx

```diff
@@ -128,7 +130,10 @@ export const InstancesTable = <T extends KafkaInstance>({
                           {row.name}
                         </Link>
                       )}
-                      isDisabled={DeletingStatuses.includes(row["status"])}
+                      isDisabled={
+                        DeletingStatuses.includes(row["status"]) ||
```

Comment:
I think we should move this logic up a level like for the other `canXYZ` callbacks. WDYT?